vip-feed-cache/class-vip-go-feed-transient.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class VIP_Go_Feed_Cache_Transient extends WP_Feed_Cache_Transient {
+class VIP_Go_Feed_Cache_Transient extends WP_Feed_Cache_Transient implements SimplePie_Cache_Base {
```

Comment:
I added the interface b/c of https://github.com/WordPress/WordPress/blob/270f2011f8ec7265c3f4ddce39c77ef5b496ed1c/wp-includes/SimplePie/Cache/Base.php#L47-L49